<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDialog</class>
 <widget class="QDialog" name="GameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="button_dialog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerName_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to Tampere! How can we call you?</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Map size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mapSize_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Large</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Small</string>
    </property>
   </item>
  </widget>
  <widget class="QGraphicsView" name="bus_graphicView">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Score Limit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="scoreLimit_spinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>700</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_dialog</sender>
   <signal>accepted()</signal>
   <receiver>GameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_dialog</sender>
   <signal>rejected()</signal>
   <receiver>GameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
